### ISSUE TITLE
[🐞] All bundles get prefetched with full priority in PrefetchServiceWorker resulting in error ERR_INSUFFICIENT_RESOURCES

### ISSUE THREAD

**Issue by DustinJSilk** [QwikDev/qwik#6397]
### Which component is affected?

Qwik Runtime

### Describe the bug

When using the PrefetchServiceWorker, the service worker prefetches all bundles with full priority. This results in all requests running in parallel without being throttled by `$maxPrefetchRequests$`.

In an app with more than ~600 bundles, many of the prefetch requests will throw an error `Failed to fetch` with `(failed)net::ERR_INSUFFICIENT_RESOURCES`

### Reproduction

https://github.com/DustinJSilk/qwik-prefetch-priority

### Steps to reproduce

Clone the reproduction url and run these commands:

`$ ./generate.sh`

`$ pnpm i`

`$ pnpm build`

Preview the app:

`$ pnpm preview`

Then in chrome
- Clear the application site data
- Refresh the page (witness no prefetching, might be another issue?)
- Refresh the page again (witness network errors `(failed)net::ERR_INSUFFICIENT_RESOURCES` and console errors)
  - If you don't see network errors, try increasing the number of pages generated in `./generate.sh`

### System Info

```shell
System:
    OS: macOS 14.4.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 474.85 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.12.2 - ~/.nvm/versions/node/v20.12.2/bin/node
    npm: 10.5.0 - ~/.nvm/versions/node/v20.12.2/bin/npm
    pnpm: 9.0.5 - ~/Library/pnpm/pnpm
  Browsers:
    Chrome: 125.0.6422.77
    Safari: 17.4.1
  npmPackages:
    @builder.io/qwik: github:QwikDev/qwik-build#55dc8da599c585a880561fe8db9dd488c043eb1c => 1.5.5-dev20240523064535 
    @builder.io/qwik-city: github:QwikDev/qwik-city-build#9306818ac6318b73b2d7b597a9e9512c2f024bc8 => 1.5.5-dev20240523065036 
    typescript: 5.4.5 => 5.4.5 
    undici: * => 6.18.1 
    vite: ^5.2.10 => 5.2.11
```

**Comment by gioboa** [comment]
Hi, I tested your repo with #6582 and it's working fine with no errors.
![image](https://github.com/QwikDev/qwik/assets/35845425/fced18ce-1438-4929-9e5a-760de7a6f2fd)
Thanks for your report.

**Comment by PatrickJS** [comment]
I'm going to close this unless we can reproduce it

**Comment by wmertens** [comment]
@gioboa did you test it on localhost or a server?

**Comment by gioboa** [comment]
Localhost
